docs/types/dynamic-dispatch.md

```diff
@@ -13,14 +13,9 @@ for usability, as users can write very flexible code that still plays nicely
 with the GUI.
 
 The current implementation of Enso supports single dispatch (dispatch purely on
-the type of `this`), but there are broader visions afoot for the final
-implementation of dynamic dispatch in Enso.
-
-> The actionables for this section include:
->
-> - Determining whether we want to support proper multiple dispatch in the
->   future. This is important to know as it has implications for the type
->   system, and the design of the dispatch algorithm.
+the type of `self`) when calling function. When calling (binary) operators Enso
+may perform more complicated dispatch when searching for the right operator
+implementation to invoke.
```

Comment:
Multiple dispatch is also used on `from` conversions, because in expression `T.from x` the function to use is based on both `T` and `x`.

Reply:
Copied in 07d353070d. Thank you.